data_model/src/lib.rs

```diff
@@ -338,7 +343,81 @@ pub enum Value {
     /// [`struct@Hash`]
     Hash(Hash),
     /// Block
-    Block(BlockValue),
+    Block(BlockValueWrapper),
+}
+
+/// Cross-platform wrapper for `BlockValue`.
+#[cfg(not(all(target_arch = "aarch64", target_os = "macos")))]
+#[derive(
+    Debug, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema, PartialOrd, Ord,
+)]
```

Comment:
I believe [`#[serde(transparent)]`](https://serde.rs/container-attrs.html#transparent) should be used here.